src/components/ErrorLogs/index.tsx

```diff
@@ -17,26 +17,26 @@ const ErrorLogs = (props: ErrorLogsProps): JSX.Element => {
         } else {
             setViewErrorLogs(!viewErrorLogs);
         }
-    }
-    const items = [{
-        key: "1",
-        label: "Logs",
-        children: (
-            <div className="log-box">
-                <pre>{errorLogs}</pre>
-            </div>
-        )
-    }];
+    };
 
     return (
-        <div>
-            <Collapse 
-                items={items}
-                activeKey={viewErrorLogs && errorLogs.length > 0 ? ["1"] : []}
-                onChange={toggleLogs}
-            />
-        </div>
+        <>
+            <Button color="primary" variant="filled" onClick={toggleLogs}>
+                Logs
+            </Button>
+            <Drawer
```

Comment:
the Drawer for logs looks great! nit: placing it at the bottom might work better for long log lines, but totally fine to keep it as is while we wait for the design